test/e2e/decryption/decryption.go

```diff
@@ -73,8 +73,8 @@ func LogsAreEqual(t *testing.T, expected, got []byte) {
 	require.NoError(t, err)
 
 	if diff := deep.Equal(expectedLines, gotLines); diff != nil {
-		t.Logf("expectedLines: %q\n", expectedLines)
-		t.Logf("gotLines: %q\n", gotLines)
+		t.Logf("expectedLines: %s\n", expected)
+		t.Logf("gotLines: %s\n", string(got))
```

Comment:
`%q` adds quotes to the string, like `"expected"`
```suggestion
		t.Logf("expectedLines: %q\n", expected)
		t.Logf("gotLines: %q\n", string(got))
```

Reply:
`expected` and `got` are JSON (with quotes), not sure if additional quotes help in this scenario, wdyt?

Reply:
this will be a bit more verbose, but makes it easier to identify start / end of the audit lines, wdyt?
```suggestion
		t.Logf("=== expected audit lines ===\n%s\n=== end expected lines ===", expected)
		t.Logf("=== actual audit lines ===\n%s\n=== end actual lines ===", string(got))
```

Reply:
sure, that works for me